docs/conf.py

```diff
@@ -70,9 +70,13 @@
 
 intersphinx_mapping = {
     'python': ('https://docs.python.org/3', None),
-    'packaging': ('https://packaging.python.org/en/latest/', None),
+    'packaging': ('https://packaging.python.org/', None),
 }
 
+# https://github.com/pypa/packaging.python.org/issues/995
+nitpick_ignore = [
+    ('py:class', 'packaging.version.Version'),  # autodoc autolinks type annotations, this one is not added
```

Comment:
This exists as https://packaging.pypa.io/en/latest/version.html#packaging.version.Version